index.csv

```diff
@@ -230,6 +236,7 @@ webpack-dev-server,MIT,0.6.0,0.6.1,0.6.2,0.7.0,0.7.1,0.7.2,0.8.0,0.8.1,0.8.2,0.9
 whatwg-fetch,MIT,0.5.0,0.6.0,0.6.1,0.7.0,0.8.0,0.8.1,0.8.2,0.9.0,0.10.0,0.10.1,0.11.0,0.11.1
 whatwg-fetch,MIT,0.7.0,0.8.0,0.8.1,0.8.2,0.9.0,0.10.0,0.10.1,0.11.0,0.11.1
 when,MIT,1.2.0,1.3.0,1.4.0,1.4.1,1.4.2,1.4.3,1.4.4,1.5.0,1.5.1,1.5.2,1.6.0,1.6.1,1.7.0,1.7.1,1.8.0,1.8.1,2.0.0,2.0.1,2.1.0,2.1.1,2.2.0,2.2.1,2.3.0,2.4.0,2.5.0,2.5.1,2.6.0,2.7.0,2.7.1,2.8.0,3.0.0,3.0.1,3.1.0,3.2.0,3.2.1,3.2.2,3.2.3,3.3.0,3.3.1,3.4.0,3.4.1,3.4.2,3.4.3,3.4.4,3.4.5,3.4.6,3.5.0,3.5.1,3.5.2,3.6.0,3.6.1,3.6.2,3.6.3,3.6.4,3.7.0,3.7.1,3.7.2,3.7.3
+when-exit,MIT,1.0.0,2.0.0,2.1.0,2.1.1,2.1.2,2.1.3
```

Comment:
Looks good. Author added license file in initial commit: https://github.com/fabiospampinato/when-exit/commit/8bb773c0c50088e2e63ee9e6354f29857b334e22